package.json

```diff
@@ -18,7 +18,10 @@
   "homepage": "https://github.com/SMH110/My-Calculator#readme",
   "devDependencies": {
     "chai": "^3.5.0",
-    "knockout": "^3.4.0",
```

Comment:
Knockout is no longer just a dev dependency. It is now an application code dependency (as we are no longer using a CDN), so we have moved it to "dependencies"